src/platforms/common/configuration/options.mdx

```diff
@@ -59,6 +59,12 @@ Enabling `debug` mode makes the SDK generate as much diagnostic data as possible
 
 </ConfigKey>
 
+<PlatformSection supported={["dotnet"]}>
+
+For app models that don't have a console to print to, such as WinForms and ASP.NET, you can <PlatformLink to="/configuration/diagnostic-logger/">customize the SDK's diagnostic logger</PlatformLink> to write to a file or to Visual Studio's debug window.
```

Comment:
Since the content isn't there for these two frameworks, I'm not sure if we should mention them. 

```suggestion
For app models that don't have a console to print to, you can <PlatformLink to="/configuration/diagnostic-logger/">customize the SDK's diagnostic logger</PlatformLink> to write to a file or to Visual Studio's debug window.
```